db/data/20230307105727_create_audits_for_existing_budgets.rb

```diff
@@ -0,0 +1,35 @@
+# Run me with `rails runner db/data/20230307105727_create_audits_for_existing_budgets.rb`
+
+# We are using the Audited Gem to create revisions for Budgets.
+#
+# The "Original" Budget will be the audit that is created when the
+# Budget is first created. However, while this works for new Budgets,
+# we must create an audit for all existing Budgets as well.
+#
+# Audited does not provide an easy way to do this. This uses a private
+# method of the library, then updates the created_at attribute to match the
+# created_at attributes of the Budget itself.
+
+Budget.class_eval do
+  def backfill_audit
+    write_audit(action: "create", audited_changes: audited_attributes,
+                comment: audit_comment)
+
+    Audited::Audit.find_by(auditable_type: "Budget", auditable_id: id).update!(created_at: created_at)
+  end
+end
+
+def budgets_without_create_audits
+  Budget
```

Comment:
Does this look okay? Is there a better way of doing it. I'm fairly sure it works as expected after poking it a few times.

Reply:
Hmmm, this is still imperfect. If someone creates a revision before this is run, we end up creating the "create" audit with the revised budget (+ the revision numbers are wrong). Can we assume that we can safely run this before any revisions are made, or do I need to do some more work to make this more robust?

<img width="1164" alt="image" src="https://user-images.githubusercontent.com/47089130/224359775-c3aeb99d-8864-4992-931b-15fc5bb83d9d.png">